ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,175 +1,180 @@
 
 With no Elastic Subnets live on Mainnet, it is clear that Permissionless Subnets as they stand today could be more desirable. There are many successful Permissioned Subnets in production but many Subnet creators have raised the above as points of concern. In summary, the Avalanche community could benefit from a more flexible and affordable mechanism to launch Permissionless Subnets.
 
+### A Note on Nomenclature
+
+Going forward, newly created networks should be referred to as L1s, or Avalanche Layer 1s. L1 validator sets created using the flow introduced in this ACP will **not** be a subset of the Primary Network validator set, making Subnets a technical misnomer. Any network created with the pre-existing flow will still be considered a Subnet, as these types of validators must stake and validate the Avalanche Primary Network.
+
+Though Subnets have always managed their own consensus mechanisms, transaction processing, and security protocols directly within their own networks, the added functionality introduced in this ACP will solidify their status as standalone networks, or layer 1 blockchains.
+
 ## Specification
 
-At a high-level, Subnets can manage their validator sets externally to the P-Chain by setting the blockchain ID and address of their _validator manager_. The P-Chain will consume Warp messages that modify the Subnet's validator set. To confirm modification of the Subnet's validator set, the P-Chain will also produce Warp messages. Subnet Validators will no longer be required to validate the Primary Network, removing the 2000 $AVAX stake requirement to be a Subnet Validator. In place of the stake requirement, a continuous fee denominated in $AVAX is introduced to maintain an active Subnet Validator. Subnet Validators are only required to sync the P-Chain (not X/C-Chain) in order to track validator set changes and support cross-Subnet communication.
+At a high-level, L1s can manage their validator sets externally to the P-Chain by setting the blockchain ID and address of their _validator manager_. The P-Chain will consume Warp messages that modify the L1's validator set. To confirm modification of the L1's validator set, the P-Chain will also produce Warp messages. L1 validators will no longer be required to validate the Primary Network, removing the 2000 $AVAX stake requirement imposed on Subnet validators. In place of the stake requirement, a continuous fee denominated in $AVAX is introduced to maintain an active Subnet validator. Subnet validators are only required to sync the P-Chain (not X/C-Chain) in order to track validator set changes and support cross-Subnet communication.
```

Comment:
Appreciate the thoroughness, I'm still going through the entire doc and plan to update it entirely so that we only use Subnet to refer to chains made pre-acp-77